src/file-download/file-list-card/file-list-card.tsx

```diff
@@ -124,6 +116,11 @@ const Component = ({ fileItem, onDownload }: FileListItemProps) => {
                 <FileSizeSection>
                     <ItemText>{fileSize ? fileSize : "-"}</ItemText>
                 </FileSizeSection>
+                {isError && (
+                    <MobileErrorMessage weight="semibold">
```

Comment:
should this go to `renderContents`? it's not showing for list card without thumbnail